alembic/versions/67109d315f17_make_classification_prob_non_nullable_.py

```diff
@@ -0,0 +1,47 @@
+"""make classification prob non-nullable, set default to 1
+
+Revision ID: 67109d315f17
+Revises: 065726baac98
+Create Date: 2022-10-21 19:25:22.336778
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '67109d315f17'
+down_revision = '065726baac98'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    classifications = sa.sql.table(
+        "classifications", sa.Column("probability", sa.VARCHAR())
+    )
+    op.execute(
+        classifications.update()
+        .where(classifications.c.probability.is_(None))
+        .values(probability=1.0)
+    )
```

Comment:
From the errors in the migration test I am going to venture a guess that some probability values are set to some string (it may be "null" or something silly like that) and then it complains it is getting a string and not a float.

Reply:
So if you figure out what those are, you may end up adding another `execute` to change that string to a float.